astro/src/content/docs/get-started/core-concepts/applications.mdx

```diff
@@ -46,6 +46,29 @@ Below is a visual reminder of the relationships between FusionAuth's primary cor
 
 <img src="/img/docs/get-started/core-concepts/core-concepts-relationships-applications.png" alt="Diagram showing Applications used within FusionAuth" />
 
+## Universal Applications
```

Comment:
I would consider moving the universal stuff to the bottom. Yes, it is shiny and new, but most viewers of this page are likely not here for this stuff